Unused type parameters in typechain/ethers-v5 6.0.1

After bumping `@typechain/ethers-v5` from `5.0.5` to `6.0.1` my project stops compiling when `noUnusedParameters` compiler option is enabled in TS 4.0.1.

The TypeChain-generated file `commons.ts` triggers an error:
```ts
contract-bindings/ethers/commons.ts:8:34 - error TS6205: All type parameters are unused.

8 export interface TypedEventFilter<EventArgsArray, EventArgsObject>
```
This is its body:
```ts
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */

import { EventFilter, Event } from "ethers";
import { Result } from "@ethersproject/abi";

export interface TypedEventFilter<EventArgsArray, EventArgsObject>
  extends EventFilter {}

export interface TypedEvent<EventArgs extends Result> extends Event {
  args: EventArgs;
}

export type TypedListener<
  EventArgsArray extends Array<any>,
  EventArgsObject
> = (
  ...listenerArg: [
    ...EventArgsArray,
    TypedEvent<EventArgsArray & EventArgsObject>
  ]
) => void;
```

--- a/packages/target-ethers-v5-test/types/commons.ts
+++ b/packages/target-ethers-v5-test/types/commons.ts
@@ -5,7 +5,7 @@
 import { EventFilter, Event } from "ethers";
 import { Result } from "@ethersproject/abi";
 
-export interface TypedEventFilter<EventArgsArray, EventArgsObject>
+export interface TypedEventFilter<_EventArgsArray, _EventArgsObject>
   extends EventFilter {}
 
 export interface TypedEvent<EventArgs extends Result> extends Event {

--- a/packages/target-ethers-v5/src/index.ts
+++ b/packages/target-ethers-v5/src/index.ts
@@ -138,7 +138,7 @@ export default class Ethers extends TsGeneratorPlugin {
     import { EventFilter, Event } from 'ethers'
     import { Result } from '@ethersproject/abi'
 
-    export interface TypedEventFilter<EventArgsArray, EventArgsObject> extends EventFilter {}
+    export interface TypedEventFilter<_EventArgsArray, _EventArgsObject> extends EventFilter {}
 
     export interface TypedEvent<EventArgs extends Result> extends Event {
       args: EventArgs;

